tests/click_tests/test_repeated_backup.py

```diff
@@ -151,6 +151,9 @@ def test_repeated_backup(
 
         second_backup_2_of_3.append(" ".join(words))
 
+    # confirm backup success
+    reset.confirm_read(debug)
```

Comment:
my vague recollection says this will break on different layout models

Reply:
68f7c1138e seems to be passing... :thinking: